worldengine/hdf5_serialization.py

```diff
@@ -0,0 +1,220 @@
+import h5py
+from worldengine.version import __version__
+from worldengine.biome import biome_name_to_index, biome_index_to_name
+from worldengine.world import World, Step
+import numpy
+
+
+def save_world_to_hdf5(world, filename):
+    f = h5py.File(filename, libver='latest', mode='w')
```

Comment:
It's not worth the work right now, but I would have liked to see an abstract class that could be sub-classed whenever a new format is added. I am fairly certain that protobuf uses almost (?) the same functions - being able to access them as `protobuf.save_world("test.world")` and `hdf5.save_world("test.world")` would seem prettier to me.^^
Maybe that's something to remember for a future PR. :)